twiggy/tests/all/expectations/dominators_wee_alloc_json

```diff
@@ -1 +1 @@
588},{"name":"type[1]: (i32, i32) -> i32","shallow_size":6,"shallow_size_percent":0.21299254526091588,"retained_size":6,"retained_size_percent":0.21299254526091588},{"name":"func[7]","shallow_size":1,"shallow_size_percent":0.03549875754348598,"retained_size":6,"retained_size_percent":0.21299254526091588,"children":[{"name":"<wee_alloc::size_classes::SizeClassAllocPolicy<'a> as wee_alloc::AllocPolicy>::min_cell_size::h6f746be886573355","shallow_size":5,"shallow_size_percent":0.1774937877174299,"retained_size":5,"retained_size_percent":0.1774937877174299}]},{"name":"func[4]","shallow_size":1,"shallow_size_percent":0.03549875754348598,"retained_size":4,"retained_size_percent":0.1419950301739439,"children":[{"name":"core::ptr::drop_in_place::h8e9fdc2437d43666","shallow_size":3,"shallow_size_percent":0.10649627263045794,"retained_size":3,"retained_size_percent":0.10649627263045794}]},{"name":"func[6]","shallow_size":1,"shallow_size_percent":0.03549875754348598,"retained_size":4,"retained_size_percent":0.1419950301739439,"children":[{"name":"core::ptr::drop_in_place::h4e5cdfd7b9310648.18","shallow_size":3,"shallow_size_percent":0.10649627263045794,"retained_size":3,"retained_size_percent":0.10649627263045794}]}]}]},{"name":"export \"hello\"","shallow_size":8,"shallow_size_percent":0.2839900603478878,"retained_size":177,"retained_size_percent":6.283280085197019,"children":[{"name":"func[8]","shallow_size":1,"shallow_size_percent":0.03549875754348598,"retained_size":169,"retained_size_percent":5.999290024849131,"children":[{"name":"hello","shallow_size":164,"shallow_size_percent":5.8217962371317,"retained_size":164,"retained_size_percent":5.8217962371317},{"name":"type[5]: () -> i32","shallow_size":4,"shallow_size_percent":0.1419950301739439,"retained_size":4,"retained_size_percent":0.1419950301739439}]}]},{"name":"export \"goodbye\"","shallow_size":10,"shallow_size_percent":0.3549875754348598,"retained_size":55,"retained_size_percent":1.9524316648917288,"children":[{"name":"func[9]","shallow_size":1,"shallow_size_percent":0.03549875754348598,"retained_size":45,"retained_size_percent":1.5974440894568689,"children":[{"name":"goodbye","shallow_size":44,"shallow_size_percent":1.5619453319133831,"retained_size":44,"retained_size_percent":1.5619453319133831}]}]},{"name":"export \"memory\"","shallow_size":9,"shallow_size_percent":0.3194888178913738,"retained_size":11,"retained_size_percent":0.3904863329783458,"children":[{"name":"memory[0]","shallow_size":2,"shallow_size_percent":0.07099751508697195,"retained_size":2,"retained_size_percent":0.07099751508697195}]},{"name":"data[0]","shallow_size":9,"shallow_size_percent":0.3194888178913738,"retained_size":9,"retained_size_percent":0.3194888178913738},{"name":"type[3]: (i32) -> nil","shallow_size":4,"shallow_size_percent":0.1419950301739439,"retained_size":4,"retained_size_percent":0.1419950301739439}]}],"summary":[{"name":"[3 Unreachable Items]","retained_size":1084,"retained_size_percent":38.480653177138805}]}
```

Comment:
the end of this file is where we can see the unreachable summary in json form. it's within a separate 'summary' object, so that we can add other little heuristics later on :)